Adds python and HuggingFace usage notes (#7)
Imageomics/LatLonCover: README.md

> This documentation summarizes the data sources used for this repository and concludes with a descripion of function outcomes.

Reviewer: ```suggestion
This documentation summarizes function usage,  data sources,  and  function outcomes.
```
